src/seq_analysis/kmer.rs

```diff
@@ -113,6 +127,39 @@ pub fn next_dnakmer(kmer: DNAKMer, k: usize, b: u8) -> DNAKMer {
     return DNAKMer(result); 
 }
 
+pub fn dna_complement(base: u8) -> u8 {
```

Comment:
You're right, that function is redundant. I don't use it anyways, when I complement I'm working on the 2 bit representation.